lib/chatbot-api/schema/schema.graphql

```diff
@@ -0,0 +1,351 @@
+}
+
+input GetRSSPostsInput {
+  workspaceId: String!
+  documentId: String!
+  lastDocumentId: String
+}
+
+type KendraIndex @aws_cognito_user_pools {
+  id: String!
+  name: String!
+  external: Boolean!
+}
+
+input ListDocumentsInput {
+  workspaceId: String!
+  documentType: String!
+  lastDocumentId: String
+}
+
+type Model @aws_cognito_user_pools {
+  name: String!
+  provider: String!
+  interface: String!
+  ragSupported: Boolean!
+  inputModalities: [String!]!
+  outputModalities: [String!]!
+  streaming: Boolean!
+}
+
+type PassageRank @aws_cognito_user_pools {
+  score: Float!
+  passage: String!
+}
+
+input QnADocumentInput {
+  workspaceId: String!
+  question: String!
+  answer: String!
+}
+
+type RagEngine @aws_cognito_user_pools {
+  id: String!
+  name: String!
+  enabled: Boolean!
+}
+
+input RankPassagesInput {
+  provider: String!
+  model: String!
+  reference: String!
+  passages: [String]!
+}
+
+input RssFeedInput {
+  workspaceId: String!
+  address: String!
+  limit: Int!
+  title: String
+  followLinks: Boolean!
+}
+
+input SemanticSearchInput {
+  workspaceId: String!
+  query: String!
+}
+
+type SemanticSearchItem @aws_cognito_user_pools {
+  sources: [String]
+  chunkId: String
+  workspaceId: ID!
+  documentId: String
+  documentSubId: String
+  documentSubType: String
+  documentType: String!
+  path: String
+  language: String
+  title: String
+  content: String
+  contentComplement: String
+  vectorSearchScore: Float
+  keywordSearchScore: Float
+  score: Float
+}
+
+type SemanticSearchResult @aws_cognito_user_pools {
+  engine: String!
+  workspaceId: String!
+  queryLanguage: String
+  supportedLanguages: [String!]
+  detectedLanguages: [DetectedLanguage!]
+  items: [SemanticSearchItem!]
+  vectorSearchMetric: String
+  vectorSearchItems: [SemanticSearchItem!]
+  keywordSearchItems: [SemanticSearchItem!]
+}
+
+type Session @aws_cognito_user_pools {
+  id: String!
+  title: String
+  startTime: AWSDateTime!
+  history: [SessionHistoryItem]
+}
+
+type SessionHistoryItem @aws_cognito_user_pools {
+  type: String!
+  content: String!
+  metadata: String
+}
+
+input TextDocumentInput {
+  workspaceId: String!
+  title: String!
+  content: String!
+}
+
+input WebsiteInput {
+  workspaceId: String!
+  sitemap: Boolean!
+  address: String!
+  followLinks: Boolean!
+  limit: Int!
+}
+
+type Workspace @aws_cognito_user_pools {
+  id: String!
+  name: String!
+  formatVersion: Int
+  engine: String!
+  status: String
+  aossEngine: String
+  languages: [String]
+  hasIndex: Boolean
+  embeddingsModelProvider: String
+  embeddingsModelName: String
+  embeddingsModelDimensions: Int
+  crossEncoderModelName: String
+  crossEncoderModelProvider: String
+  metric: String
+  index: Boolean
+  hybridSearch: Boolean
+  chunkingStrategy: String
+  chunkSize: Int
+  chunkOverlap: Int
+  vectors: Int
+  documents: Int
+  sizeInBytes: Int
+  kendraIndexId: String
+  kendraIndexExternal: String
+  kendraUseAllData: Boolean
```

Comment:
`String` type is preventing users from adding new data sources for Kendra workspaces. Changing this to `Boolean` it seems to fix the issue.

Ref: https://github.com/massi-ang/aws-genai-llm-chatbot/blob/0ace107fb9db1ba0ad086d6114a678675162b263/lib/user-interface/react-app/src/pages/rag/add-data/add-data.tsx#L160-L171

```suggestion
  kendraIndexExternal: Boolean
```